content/reference/waPC/ffi.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "waPC FFI Functions"
+date: 2018-12-29T11:02:05+06:00
+weight: 6
+draft: false
+---
+For the curious, here is the exact list of the waPC protocol’s functions. The guest always imports from a module called “wapc”, and all function names are preceded by two underscores to avoid accidental collisions with other exported functions.
+
+| Function | Owner | Description |
```

Comment:
These function and owner columns end up compressing to a pretty small width, is there a way to enforce a minimum width here?
![image](https://user-images.githubusercontent.com/12040685/109194630-9d571980-7767-11eb-9c86-8be4f33d647e.png)

Reply:
I don't know how to fix this. I've tried playing with the markdown, but I think it might be something related to the CSS that comes with the default site template. Thoughts @billyoung or @ChrisRx ?

Reply:
If you can open up the HTML in the chrome tools and adjust the CSS so it fits, then we can figure out how to make those changes in the real CSS

Reply:
There's probably a few ways to do this, the initial way that I solved it was by removing the work-break for tables. Not sure if this has other consequences:
```
.content * {
  /* word-break: break-word; */
  overflow-wrap: break-word;
  }
```
According to https://stackoverflow.com/questions/36121672/set-table-column-width-via-markdown, we can either wrap the table in a `div` and give it our own custom styles to keep the table isolated from the site CSS, or we can use `&nbsp` repeatedly to set a width.